packages/atst/src/README.md

```diff
@@ -0,0 +1,35 @@
+# src
```

Comment:
What is the difference between a README.md in the src and one in the root? Are we going to include this when we build and publish on NPM?

Reply:
The readmes in the folder only explain what is happening in that folder.   This one simply explains what every file and folder is.    It's pretty nice on github since the readme shows up when you click on the folder. It would show up on NPM too unless we started explicitly adding files to package.json.  Something we could do but seems unecessary